cumulusci/tasks/salesforce/EnsureRecordTypes.py

```diff
@@ -116,11 +120,19 @@ def _build_package(self):
                         self.options["sobject"]
                     )
                 )
+                # Lead, Case, and Solution require a default value in their Business Process
+                # Opportunity prohibits it.
+                if self.options["sobject"] == "Opportunity":
+                    default = "false"
+                else:
+                    default = "true"
```

Comment:
💭 :
```python
       default = self.options["sobject"] != "Opportunity"
```

Then again, the existing version is probably more readable.

Reply:
Yeah, I was expecting more branches here and then discovered Case, Lead, and Solution all play by the same rules, but Opportunity doesn't. Adopting your proposal.

Reply:
(Not before I make a couple of bone-headed mistakes, of course)